Cargo.lock

```diff
@@ -644,6 +653,19 @@ dependencies = [
  "encoding_rs",
 ]
 
+[[package]]
+name = "chrono"
+version = "0.4.42"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "145052bdd345b87320e369255277e3fb5152762ad123a901ef5c262dd38fe8d2"
+dependencies = [
+ "iana-time-zone",
+ "js-sys",
+ "num-traits",
+ "wasm-bindgen",
+ "windows-link 0.2.0",
+]
```

Comment:
I'd like to add Jiff support upstream if we can, possibly before merging this \cc @BurntSushi

Reply:
From a very quick glance, it's probably non-trivial for them to support both. Depending.

Anywho, I opened an issue: https://github.com/apache/opendal-reqsign/issues/631